config/index.js

```diff
@@ -5,67 +5,152 @@ module.exports = {
   },
   rules: {
     'accessor-pairs': 'error',
+    'array-bracket-newline': 'off',
+    'array-bracket-spacing': ['error', 'never'],
+    'array-callback-return': 'error',
+    'array-element-newline': 'off',
+    'arrow-body-style': 'off',
+    'arrow-parens': 'error',
     'arrow-spacing': ['error', { 'before': true, 'after': true }],
+    'block-scoped-var': 'error',
     'block-spacing': ['error', 'always'],
     'brace-style': 'error',
+    'callback-return': 'error',
     'camelcase': ['error', { 'properties': 'never', 'allow': ['^UNSAFE_'] }],
+    'capitalized-comments': 'off',
+    'class-methods-use-this': 'off',
     'comma-dangle': ['error', 'always-multiline'],
     'comma-spacing': ['error', { 'before': false, 'after': true }],
     'comma-style': ['error', 'last'],
+    'complexity': 'off',
+    'computed-property-spacing': 'error',
+    'consistent-return': 'error',
+    'consistent-this': 'error',
```

Comment:
Might it be better to use `self` as the designated `this` alias? That seems to be what we've used traditionally - I've never seen `that` in our codebase.